vacationtime-solution/src/Code.js

```diff
@@ -0,0 +1,195 @@
+      numRows, 1);
+  var dropdownValues = ['APPROVED', 'NOT APPROVED', 'IN PROGRESS'];
+  var rule = SpreadsheetApp.newDataValidation().requireValueInList(dropdownValues)
+      .build();
+  approvalColumnRange.setDataValidation(rule);
+  approvalColumnRange.setValue('IN PROGRESS');
+}
+
+/** 
+ * Creates Notified Column.
+ */
+function createNotifiedColumn() {
+  var sheet = SpreadsheetApp.getActiveSheet();
+  var lastRow = sheet.getLastRow();
+  var lastCol = sheet.getLastColumn();
+  var frozenRows = sheet.getFrozenRows();
+  var startRow = frozenRows + 1;
+  var numRows = lastRow - frozenRows;
+  
+  // Sets up column properties.
+  sheet.insertColumnAfter(lastCol);
+  sheet.getRange(1, COLUMN_NUMBER.NOTIFIED)
+      .setValue('NOTIFIED STATUS');
+  var notifiedValues = ['NOTIFIED', 'NOT NOTIFIED'];
+  var rule = SpreadsheetApp.newDataValidation().requireValueInList(notifiedValues)
+      .build();
+
+  // Sets column's cells to be drop-down menus.
+  var notifiedColRange = sheet.getRange(startRow, COLUMN_NUMBER.NOTIFIED,
+      lastRow - frozenRows, 1);
+  notifiedColRange.setDataValidation(rule);
+  notifiedColRange.setValue('NOT NOTIFIED'); 
+  return notifiedColRange;
+}
+
+/**
+ * Creates a calendar event for an employee whose vacation
+ * request has been approved.
+ * @param {String} managerEmail Manager's email.
+ * @param {String} employeeName Name of employee.
+ * @param {String} employeeEmail Email of employee.
+ * @param {date} startDate Vacation request start date.
+ * @param {date} endDate Vacation request end date.
+**/
+function createCalEvent(managerEmail, employeeName, employeeEmail,
+                       startDate, endDate) { 
+    var managerCal = CalendarApp.getCalendarById(managerEmail);
+    // Creates a calendar event.
+    var event = managerCal.createEvent('APPROVED VACATION TIME FOR ' +
+        employeeName, startDate, endDate, {
+        description: 'Your vacation time from ' +
+                                            startDate + ' to ' + endDate +
+                                            ' has been approved! Enjoy!',
+        guests: employeeEmail,
+        sendInvites: true,
+    });
+}
+
+/**
+ * Sends e mails to employees whose vacation time request
+ * was NOT approved.
+ * @param {String} employeeEmail Email of employee.
+**/
+function sendEmails(employeeEmail) {
+  MailApp.sendEmail(employeeEmail, REJECTION_EMAIL_SUBJECT, 
+                        REJECTION_EMAIL_MESSAGE);
+}
+
+/**
+ * Checks the approval status of each employee and notifies
+ * them of their status accordingly.
+ * @param {String} employeeEmail Email of employee.
+ * @param {String} managerEmail Manager's email.
+ * @param {String} employeeName Name of employee.
+ * @param {String} approvalStatus Manager-set status.
+ * @param {date} startDate Vacation request start date.
+ * @param {date} endDate Vacation request end date.
+**/
+function checkApproval(employeeEmail, managerEmail, employeeName, 
```

Comment:
Suggest either renaming or refactoring. I expect checkApproval() to do just that -- check if it is approve.d But this is doing a lot more than that.

Either rename to something like 'checkApprovalAndNotify' or split this up in to different functions so that each has a clear purpose.